api/src/opentrons/protocol_engine/state/pipettes.py

```diff
@@ -189,3 +201,15 @@ def get_is_ready_to_aspirate(
             self.get_aspirated_volume(pipette_id) > 0
             or pipette_config["ready_to_aspirate"]
         )
+
+    def get_attached_tip_labware_id(self, pipette_id: str) -> str:
+        """Get the tiprack labware id of the attached tip."""
+        tipracks_by_pip_id = self.get_attached_tip_labware_by_id()
+        labware_id = tipracks_by_pip_id.get(pipette_id)
+        if labware_id:
```

Comment:
```suggestion
        if labware_id is not None:
```

In general, I think when we use string IDs in `if` statements, we should use `if ___ is not None` instead if `if ___`. Otherwise, we'll accidentally treat empty strings (`""`) as null IDs.

This might be moot depending on how my `PipetteTipInfoNotFoundError` comment below is addressed.